ruby/person-records.json

```diff
@@ -0,0 +1,5 @@
+[
```

Comment:
This json file and the `hihello.txt` are going to be exactly the same in each language directory. I thought we could keep them in a separate folder so not to copy paste them over and over, but it looks like the docker build can only copy files that are in the build context (the python or ruby folders, currently).

Reply:
Hmm, I never thought of that. I think it's fine for now.

I've been brainstorming ways for the future that we can put the contents of the file in the test method itself. I haven't quite figured out a good way to do it, but it could look something like this:

```python
def test_read_json():
  docker_runner.add_test_file("person-records.json", """
    [
      {"first_name": "Bob", "last_name": "Barker", "age": 84 },
      {"first_name": "Tina", "last_name": "Turner", "age": 67 },
      {"first_name": "Steven", "last_name": "Sagal", "age": 98 }
    ]
  """
  )
  docker_runner.run(*language.script("read_json"), "person-records.json")
  #... rest of test as we currently have it
  
```